text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+Clarify API description namespace and Iron out issues which rose during API Blueprint serialization to API description namespace.
+
+# Motivation
+The purpose of this document is to clear inconsistencies in the format and make it easier to build tools around it.
+
+# Detailed design
+
+## HREF Variables
+We need to clarify about the value of the `content` in the `hrefVariables` element.
+
+## Data Structure
+We don't need a `Data Structure` because an element which is an `object`, `extend` or `array` can be considered as a data structure element.
+
+*(The current spec is a lot confusing. What is the `Data Structure Element`? Why is `Data Structure` inheriting from it?)*
+
+## Resource
+Since `hrefVariables` is an element, we should move it to be under `content` rather than leave it under `attributes`.
+
+## Transition
+Similarily to the Resource, we need to move `hrefVariables` to be under `content`. `Data Structure` in Resource is under `content`, but it is under `attributes` in Transition. To resolve this inconsistency, we need to move `Data Structure` to be under `content`. Another reason for it is that `Data Structure` is an element.
```

Comment:
I feel there is a couple of things mixed here: 

:-1:  to "we need to move `hrefVariables` to be under `content` (see above)"

> `Data Structure` in Resource is under `content`, but it is under `attributes` in Transition.

There is fundamental difference between the two – data structure in Resource describes the content of a resource. While in transition the data structure describes input parameters (attributes) of the transition. 

:-1: to "To resolve this inconsistency, we need to move `Data Structure` to be under `content`."

and as mentioned above "Another reason for it is that `Data Structure` is an element." is not a reason for thing to be in content.